### ISSUE TITLE
[Functional-Social Embedded-Social Activities]: Incorrect time is displaying for every posted topic (Comment, Reply or any other topic).

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#23]
Priority: 

Repro Steps
1. Launch the One Bus Away App.
2. Navigate to ""Sign In"" link under Hamburger link and press ENTER.
3. Sign in screen showing all currently supported identity service providers will appear.
4. Use any service provider for log in.
5. Select any location bubble and see the given arrivals.
6. Navigate to ""Join Discussion"" button and press ENTER.
7. Create any profile and comment on that profile post.
8. Navigate to ""reply"" button and press ENTER to reply on that comment.
9. Verify the posted time for ""Comment"" ""Reply"" and ""Discussion"".

Note: 
1> Same issue is repro when user ""reply comment"".
2> Same issue is repro when user ""Join Discussion"".

Expected
Correct posted time should be display for the every posted ""comment"", ""reply"" comment as well as""join discussion"" and this time should also get updated timely. So that user gets to know when any particular comment/reply was posted.

Actual
When user submit any comment then the posted time is displaying ""9h"" by default. Initially it is displaying ""0s"" after 1 second it changes to ""9h"".

MAS Rule: Functional

Impact
Users will get confused and face difficulty if the incorrect posted time is displayed for comment/reply/post.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
